src/lib/appContainer.ts

```diff
@@ -67,46 +67,52 @@ export interface IAppContainer {
   accountHolderApiGateway: IAccountHolderApiGateway;
 }
 
-export const appContainer: IAppContainer = {
-  /* Simple use cases */
-  getCachedRegistration: getCachedRegistration,
-  clearCachedRegistration: clearCachedRegistration,
-  getSession: getSession,
+// "override" is spread over the default appContainer at the bottom of this method to enable injecting mocks et al.
+export const getAppContainer = (override?: IAppContainer): IAppContainer => {
```

Comment:
pick a better name for the param please 😋

Reply:
Maybe just `overrides` would be a bit clearer? Still feels a little clunky though

Reply:
going with Issy's suggestion